api/backend.go

```diff
@@ -485,6 +486,52 @@ func (b *StatusBackend) SendDataNotification(dataPayloadJSON string, tokens ...s
 	return err
 }
 
+func (b *StatusBackend) InjectChatAccount(chatKeyHex, encryptionKeyHex string) error {
```

Comment:
Missing comment :) Also wondering if we should keep `Inject` here or change to `Set` 🤔